code/_helpers/time.dm

```diff
@@ -16,89 +16,10 @@
 		wtime = world.time
 	return time2text(wtime - GLOB.timezoneOffset + (12 * 36000), format)
 
-#define TimeOfGame (get_game_time())
-#define TimeOfTick (world.tick_usage*0.01*world.tick_lag)
-/proc/get_game_time()
-	var/global/time_offset = 0
-	var/global/last_time = 0
-	var/global/last_usage = 0
-
-	var/wtime = world.time
-	var/wusage = world.tick_usage * 0.01
-
-	if(last_time < wtime && last_usage > 1)
-		time_offset += last_usage - 1
-
-	last_time = wtime
-	last_usage = wusage
-
-	return wtime + (time_offset + wusage) * world.tick_lag
-
-var/roundstart_hour
-var/station_date = ""
-var/next_station_date_change = 1 DAY
-
-#define duration2stationtime(time) time2text(station_time_in_ticks + time, "hh:mm")
-#define worldtime2stationtime(time) time2text(roundstart_hour HOURS + time, "hh:mm")
-#define round_duration_in_ticks (round_start_time ? world.time - round_start_time : 0)
-#define station_time_in_ticks (roundstart_hour HOURS + round_duration_in_ticks)
-
-/proc/stationtime2text()
-	return time2text(station_time_in_ticks, "hh:mm")
-
-/proc/stationdate2text()
-	var/update_time = FALSE
-	if(station_time_in_ticks > next_station_date_change)
-		next_station_date_change += 1 DAY
-		update_time = TRUE
-	if(!station_date || update_time)
-		var/extra_days = round(station_time_in_ticks / (1 DAY)) DAYS
-		var/timeofday = world.timeofday + extra_days
-		station_date = num2text((text2num(time2text(timeofday, "YYYY"))+544)) + "-" + time2text(timeofday, "MM-DD")
-	return station_date
-
-
-/* Returns 1 if it is the selected month and day */
-proc/isDay(var/month, var/day)
-	if(isnum(month) && isnum(day))
-		var/MM = text2num(time2text(world.timeofday, "MM")) // get the current month
-		var/DD = text2num(time2text(world.timeofday, "DD")) // get the current day
-		if(month == MM && day == DD)
-			return 1
-
-		// Uncomment this out when debugging!
-		//else
-			//return 1
-
-var/next_duration_update = 0
-var/last_round_duration = 0
-var/round_start_time = 0
-
-/hook/roundstart/proc/start_timer()
-	round_start_time = world.time
-	return 1
-
-/proc/roundduration2text()
```

Comment:
we were talking about this recently, don't we need this proc?

Reply:
> we were talking about this recently, don't we need this proc?

Check define ROUND